README.md

```diff
@@ -1,70 +1,113 @@
+<p>The player will use the dinosaur legend on each side of the battlefield to understand the point value of a specific dinosaur.
+We encourage the player to group like dinosaurs together when adding up the totals. Using your imagination with addition or multiplication will make adding their values easier.</p>
+   
+Be careful! The game will add in subtraction dinosaurs as the difficulty increases.
+   
+<p>The player will adjust the input field to equal each teams total point value.
+   Once you are certain of your answer click Battle!</p>
 
-## Learn More
+---
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+**How is the score calculated?**
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+Once the user starts the battle, the game will count each dinosaurs value starting with the red team.
+The computer will place each teams total value in the actual count container.
+If atleast one of your answers is equal to the actual count value, you will be allowed to proceed to the next level.
+A perfect game will consist of two correct answers awarding you a badge!
+You can view your earned badge on the level select screen. ~To be implemetned
 
-### Code Splitting
+---
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+**Difficulty settings**
 
-### Analyzing the Bundle Size
+Beginner mode: Maximum of three dino types for each team.
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+Average mode: Maximum of four dino types for each team.
 
-### Making a Progressive Web App
+Hard mode: Maximum of four dino types for each team, fossil dinos added for subtraction.
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+</details>
 
-### Advanced Configuration
+## Summary
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+To be written.
 
-### Deployment
+## Authors
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+| 👤 **Tim Jackson**                                           | 👤 **savwiley**                                 | 👤 **rankoliang**                                   |
```

Comment:
Ah, this is nice! This isn't part of the review, but my twitter is `https://twitter.com/savwiley` and my linkedin is `https://www.linkedin.com/in/savwiley`.

Reply:
added.